terracotta-kit/src/assemble/server/bin/start-tc-server.sh

```diff
@@ -54,48 +54,46 @@ function setPluginClasspath {
 
     for pluginDir in "${PLUGIN_LIB_DIR}" "${PLUGIN_API_DIR}"
     do
-        if [ -e "${pluginDir}" ]
+        if [ -d "${pluginDir}" ]
         then
             for jarFile in "${pluginDir}"/*.jar
             do
-                PLUGIN_CLASSPATH=${PLUGIN_CLASSPATH}:${jarFile}
-            done
-#  Adding SLF4j libraries to the classpath of the server to 
-#  support services that may use SLF4j for logging
-            for jarFile in "${TC_SERVER_DIR}"/lib/slf4j*.jar
-            do
-                PLUGIN_CLASSPATH=${PLUGIN_CLASSPATH}:${jarFile}
+                PLUGIN_CLASSPATH="${PLUGIN_CLASSPATH}:${jarFile}"
             done
+
         fi
     done
+    #  Adding SLF4j libraries to the classpath of the server to
```

Comment:
do we even need to add slf4j api jar explicitly, looks like this gets added to classpath via tc.jar already 

````
Class-Path: dso-l2-5.3-SNAPSHOT.jar tc-config-parser-10.3.0-pre15.jar
 tcconfig-schema-10.3.0-pre15.jar logback-classic-1.1.3.jar logback-co
 re-1.1.3.jar slf4j-api-1.7.7.jar entity-server-api-1.3.0-pre12.jar en
 tity-common-api-1.3.0-pre12.jar standard-cluster-services-1.3.0-pre12
 .jar packaging-support-1.3.0-pre12.jar monitoring-support-1.3.0-pre12
 .jar common-5.3-SNAPSHOT.jar tc-messaging-5.3-SNAPSHOT.jar build-data
 -5.3-SNAPSHOT.jar management-5.3-SNAPSHOT.jar connection-api-1.3.0-pr
 e12.jar commons-cli-1.1.jar resources/
````